test/powershell/Host/ConsoleHost.Tests.ps1

```diff
@@ -1,5 +1,7 @@
 using namespace System.Diagnostics
 
+Import-Module $PSScriptRoot\..\Common\Test.Helpers.psm1
```

Comment:
Clear.
I have a couple more questions.
1. We have a few more helpers modules (ex., for parser). Should we move them also in the new shared folder?
2. Can we make these modules as submodules general module and load all the modules with a single command? What is the point of importing them in multiple test files?